This crate depends on libc, but doesn't declare that dependency

It refers to `atexit` in particular.

Unless some other crate pulls libc accidentally you get a linking error like this when using this crate.
```
[INFO] [stdout]   = note: /usr/bin/ld: /opt/rustwide/target/debug/deps/liblibmimalloc_sys-1551225da21123ef.rlib(static.o): in function `mi_process_load':
[INFO] [stdout]           /opt/rustwide/cargo-home/registry/src/github.com-1ecc6299db9ec823/libmimalloc-sys-0.1.26/c_src/mimalloc/src/init.c:533: undefined reference to `atexit'
[INFO] [stdout]           collect2: error: ld returned 1 exit status
```

This was found when testing some linking order changes in `rustc` in https://github.com/rust-lang/rust/pull/102832#issuecomment-1279772306 (build failure - https://crater-reports.s3.amazonaws.com/pr-102832-1/try%230c94de7c1a8f4b82b92d562847c35f513f520c7a/reg/mimalloc-0.1.30/log.txt).

--- a/libmimalloc-sys/Cargo.toml
+++ b/libmimalloc-sys/Cargo.toml
@@ -12,6 +12,7 @@ links = "mimalloc"
 
 [dependencies]
 cty = { version = "0.2", optional = true }
+libc = "0.2"
 
 [build-dependencies]
 cc = "1.0"

--- a/libmimalloc-sys/src/lib.rs
+++ b/libmimalloc-sys/src/lib.rs
@@ -3,6 +3,8 @@
 
 use core::ffi::c_void;
 
+extern crate libc;
+
 #[cfg(feature = "extended")]
 mod extended;
 #[cfg(feature = "extended")]

